src/main/java/it/pn/frontend/e2e/section/destinatario/personaGiuridica/AggiungiDelegaPGSection.java

```diff
@@ -55,9 +64,17 @@ public AggiungiDelegaPGSection(WebDriver driver) {
     public void waitLoadAggiungiDelegaPage() {
         try {
             By titlePageBy = By.id("Aggiungi una delega-page");
-            this.getWebDriverWait(40).withMessage("Il titolo della pagina aggiungi delegha non è caricato").until(ExpectedConditions.visibilityOfElementLocated(titlePageBy));
-            this.getWebDriverWait(40).withMessage("il campo codice fiscale non è visibile").until(ExpectedConditions.visibilityOf(this.codiceFiscaleInput));
-            this.getWebDriverWait(40).withMessage("Il bottone persona giuridica non è visibile").until(ExpectedConditions.visibilityOf(this.personaGiuridicaButton));
+            this.getWebDriverWait(10).withMessage("Il titolo della pagina aggiungi delegha non è caricato").until(ExpectedConditions.visibilityOfElementLocated(titlePageBy));
+            this.getWebDriverWait(10).withMessage("Il bottone persona giuridica non è visibile").until(ExpectedConditions.visibilityOf(this.personaGiuridicaButton));
+            this.getWebDriverWait(10).withMessage("Il campo nome persona fisica non è visibile").until(ExpectedConditions.or(ExpectedConditions.visibilityOfAllElements(this.nomeInput, this.cognomeInput), ExpectedConditions.visibilityOf(this.ragioneSocialeInput)));
+            this.getWebDriverWait(10).withMessage("il campo codice fiscale non è visibile").until(ExpectedConditions.visibilityOf(this.codiceFiscaleInput));
+            By radioButtonTuttiEnti = By.id("tutti-gli-enti-selezionati");
+            this.getWebDriverWait(10).withMessage("Il radio button tutti gli enti non è visibile").until(ExpectedConditions.visibilityOfElementLocated(radioButtonTuttiEnti));
+            By radioButtonSoloEnti = By.id("enti-selezionati");
+            this.getWebDriverWait(10).withMessage("Il radio button solo enti selezionati non è visibile").until(ExpectedConditions.visibilityOfElementLocated(radioButtonSoloEnti));
+            this.getWebDriverWait(10).withMessage("Il radio button tutti gli enti non è visibile").until(ExpectedConditions.visibilityOf(this.dataTermineDelegaInput));
```

Comment:
This is needed because you need to verify if the date is tomorrow
```suggestion

            LocalDate dateTomorrow = LocalDate.now().plusDays(1);
            DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd/MM/yyyy");
            this.getWebDriverWait(10).withMessage("Il campo periodi di validità della delega non è visibile o non ha la data corretta").until(ExpectedConditions.and(
                    ExpectedConditions.visibilityOf(this.dataTermineDelegaInput),
                    ExpectedConditions.attributeContains(this.dataTermineDelegaInput, "value", dateTomorrow.format(formatter))
            ));
            
```